README.md

```diff
@@ -1 +1,82 @@
-# HELLO_MICROVERSE-2023
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Table of Contents
+
+  * About the Project
+   - Built With
+
+  *Getting Started
+    - Prerequisites
+    - Setup
+    - Install
+    - Usage
+    - Run test
+    - Deployment
+
+  * Authors
+
+  * Contributing
+
+  * Show your support
+
+  * Acknowledgements
+
+  * License
+
+# HELLO MICROVERSE
+
+> My first project at microverse with a simple message "Hello Microverse"
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+To get a local copy up and running on your machine, follow these simple steps:
+Prerequisites
+In order to run this project you need:
+
+Setup
+Clone this repository to your desired folder:
+
+Install
+Install this project with:
+
+Usage
+To run the project, execute the following command:
+
+Run tests
+To run tests, run the following command:
+
+Deployment
+You can deploy this project using:
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: (https://github.com/3nityGitHub)
+- Twitter: (https://twitter.com/tbarijohnson)
+
```

Comment:
- [ ] Please add a `Future Features` section to your project. You can add as future features things that can be added to improve the project (exemple: `- Add multiple header`). This will help demonstrate a growth mindset 🌱 😺